src/parse.c

```diff
@@ -1518,6 +1518,8 @@ static const mapping_entry_handler routes_handlers[] = {
     {"via", YAML_SCALAR_NODE, handle_routes_ip, NULL, route_offset(via)},
     {"metric", YAML_SCALAR_NODE, handle_routes_guint, NULL, route_offset(metric)},
     {"mtu", YAML_SCALAR_NODE, handle_routes_guint, NULL, route_offset(mtubytes)},
+    {"initcwnd", YAML_SCALAR_NODE, handle_routes_guint, NULL, route_offset(initcwnd)},
+    {"initrwnd", YAML_SCALAR_NODE, handle_routes_guint, NULL, route_offset(initrwnd)},     
```

Comment:
```suggestion
    {"initrwnd", YAML_SCALAR_NODE, handle_routes_guint, NULL, route_offset(initrwnd)},
```
Please fix trailing whitespace.